.github/workflows/ubuntu_clean_cmake_build.yml

```diff
@@ -0,0 +1,22 @@
+name: Minimal CMake build in Ubuntu
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-python@v1
+    - name: install minimal requirements
+      run: sudo apt-get update && sudo apt-get install -y libglib2.0-dev libmosquitto-dev libflatbuffers-dev flatbuffers-compiler
+    - name: run CMake
+      run: cmake . && make -j 2
+
+# TODO: get libmosquitto's pkgconfig for Ubunto or write a workaround for non-pkgconfig.
```

Comment:
As I know, the libmosquitto's pkgconfig is included in libmosquitto-dev.

Reply:
github-action script updated and tested: https://github.com/myungjoo/aitt/runs/5861639987?check_suite_focus=true

(libmosquitto-dev pkgconfig is available since 20.04. updated the comment)